plugins/blip/modalday/SMBGBox.js

```diff
@@ -11,6 +16,145 @@ d3.chart('SMBGBoxOverlay', {
     var boxPlotsGroup = this.base.insert('g', '#modalDays').attr('id', 'overlayUnderneath');
     var meanCirclesGroup = this.base.append('g').attr('id', 'overlayOnTop');
 
+    function getMsPer24(d) {
+      return dt.getMsPer24(d.msX);
+    }
+
+    /**
+     * Get the coordinates of the mean point given a record
+     * 
+     * @param  {Object} d the record
+     * @return {Array}   coordinates
+     */
+    function getMeanPosition(d) {
+      var mean = d3.select('#meanCircle-'+d.id);
+      return [mean.attr('cx'), mean.attr('cy')];
+    }
+
+    var tooltipHtml = function(tooltip, d) {
+      var rectOpts = chart.opts().infoRects;
+      var xScale = chart.xScale();
+      var yScale = chart.yScale();
+
+      var parent = d3.select(tooltip.tooltipGroup.node().parentNode);
+
+      var rangeLabels = parent.append('g')
+        .attr('class', 'smbgRangeInfo modalDay');
+
+      var maxYRect = yScale(d.max) - rectOpts.height - rectOpts.pad;
+      var maxYText = yScale(d.max) - (rectOpts.height/2);
+      var minYRect = yScale(d.min) + rectOpts.pad;
+      var minYText = yScale(d.min) + rectOpts.height;
+
+      if (d.max > 385) {
```

Comment:
I'm in the process of checking, but I believe I did. 

This code just makes the label overlay the box instead of being above or below is respectively. Is there a better strategy for giving space to these labels so that they can always be above or below?